exhibitor-standalone/src/main/resources/buildscripts/standalone/gradle/build.gradle

```diff
@@ -1,7 +1,17 @@
+version = '1.0'
```

Comment:
Ok given what this is for. It looks fine set the version to what makes the most sense auto calculation won't happen in here.

LGTM.